packages/react-components/react-card/src/components/Card/useCardStyles.ts

```diff
@@ -175,6 +219,12 @@ const useStyles = makeStyles({
       ...shorthands.borderColor(tokens.colorTransparentStroke),
     },
   },
+
+  select: {
+    position: 'absolute',
+    top: `var(${cardCSSVars.cardSizeVar})`,
+    right: `var(${cardCSSVars.cardSizeVar})`,
```

Comment:
Griffel handles RTL.

Reply:
AFAIK it's independent. Any `*right` gets converted to a `*left` and vice-versa.